README.md

```diff
@@ -123,21 +123,21 @@ Adopters
 
 | Company | Logo | Industry | Use of Gensim |
 |---------|------|----------|---------------|
-| [RARE Technologies](https://rare-technologies.com/) | ![rare](docs/src/readme_images/rare.png) | ML & NLP consulting | Creators of Gensim – this is us! |
-| [Amazon](http://www.amazon.com/) |  ![amazon](docs/src/readme_images/amazon.png) | Retail |  Document similarity. |
-| [National Institutes of Health](https://github.com/NIHOPA/pipeline_word2vec) | ![nih](docs/src/readme_images/nih.png) | Health | Processing grants and publications with word2vec. |
-| [Cisco Security](http://www.cisco.com/c/en/us/products/security/index.html) | ![cisco](docs/src/readme_images/cisco.png) | Security |  Large-scale fraud detection. |
-| [Mindseye](http://www.mindseyesolutions.com/) | ![mindseye](docs/src/readme_images/mindseye.png) | Legal | Similarities in legal documents. |
-| [Channel 4](http://www.channel4.com/) | ![channel4](docs/src/readme_images/channel4.png) | Media | Recommendation engine. |
-| [Talentpair](http://talentpair.com) | ![talent-pair](docs/src/readme_images/talent-pair.png) | HR | Candidate matching in high-touch recruiting. |
-| [Juju](http://www.juju.com/)  | ![juju](docs/src/readme_images/juju.png) | HR | Provide non-obvious related job suggestions. |
-| [Tailwind](https://www.tailwindapp.com/) | ![tailwind](docs/src/readme_images/tailwind.png) | Media | Post interesting and relevant content to Pinterest. |
-| [Issuu](https://issuu.com/) | ![issuu](docs/src/readme_images/issuu.png) | Media | Gensim's LDA module lies at the very core of the analysis we perform on each uploaded publication to figure out what it's all about. |
-| [Search Metrics](http://www.searchmetrics.com/) | ![search-metrics](docs/src/readme_images/search-metrics.png) | Content Marketing | Gensim word2vec used for entity disambiguation in Search Engine Optimisation. |
-| [12K Research](https://12k.com/) | ![12k](docs/src/readme_images/12k.png)| Media |   Document similarity analysis on media articles. |
-| [Stillwater Supercomputing](http://www.stillwater-sc.com/) | ![stillwater](docs/src/readme_images/stillwater.png) | Hardware | Document comprehension and association with word2vec. |
-| [SiteGround](https://www.siteground.com/) |  ![siteground](docs/src/readme_images/siteground.png) | Web hosting | An ensemble search engine which uses different embeddings models and similarities, including word2vec, WMD, and LDA. |
-| [Capital One](https://www.capitalone.com/) | ![capitalone](docs/src/readme_images/capitalone.png) | Finance | Topic modeling for customer complaints exploration. |
+| [RARE Technologies](https://rare-technologies.com/) | ![rare](https://raw.githubusercontent.com/piskvorky/gensim/develop/docs/src/readme_images/rare.png) | ML & NLP consulting | Creators of Gensim – this is us! |
```

Comment:
Why the absolute links? The README will now no longer render except with internet.

All the referenced files are local, and IMO should be read as local. This is not a "github README", it's a project README, which currently happens to be hosted on github.

Reply:
when you publish this package to PyPi, it won't have the images from the docs folder bundled along with it, so when the PyPi page renders the long description text, it won't know what `docs/src/readme_images`, so no images will appear. here's a [stackoverflow question](https://stackoverflow.com/questions/41983209/how-do-i-add-images-to-a-pypi-readme-that-works-on-github) on the subject.

perhaps as a middle ground solution we can run a `.replace(DOC_IMAGES_FOLDER, RAW_GIT_URL)` on the README text in setup.py

Reply:
ok, I've pushed up an alternate approach. did my best to replicate the existing code/comment style but obviously feel free to request any change or edit the code to your liking.

Reply:
Ah that makes sense, thank you.

Your solution is great I think, but let's hear from @mpenkov :)